src/app/api/services/dot-localstorage/dot-localstorage.service.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { TestBed } from '@angular/core/testing';
+
+import { DotLocalstorageService } from './dot-localstorage.service';
+
+describe('DotLocalstorageService', () => {
+    let service: DotLocalstorageService;
+    beforeEach(() => {
+        TestBed.configureTestingModule({});
+
+        service = TestBed.get(DotLocalstorageService);
+    });
+
+    describe('setItem', () => {
+        beforeEach(() => {
+            spyOn(window.localStorage, 'setItem');
+        });
+
+        it('should set string', () => {
+            service.setItem<string>('hello', 'world');
+            expect(window.localStorage.setItem).toHaveBeenCalledWith('hello', 'world');
```

Comment:
I think here is better do: localStorage.getItem('hello')

Reply:
What?